src/libraries/System.Text.Json/src/PACKAGE.md

```diff
@@ -1,11 +1,258 @@
 ## About
 
+<!-- A description of the package and where one can find more documentation -->
```

Comment:
What's the significance of XML remarks? Providing a template for other packages?

Reply:
Yes, this is the template that will remain.